iosApp/widget/CommuteDepartureBoardView.swift

```diff
@@ -0,0 +1,433 @@
+                title: station.displayName,
+                destinationStation: entry.configuration.destinationStation.toStation()?.displayName,
+                textColor: textColor
+            )
+            
+            if !station.trains.isEmpty {
+                CommuteTrainDisplay(
+                    trains: Array(station.trains.prefix(6)),
+                    entry: entry,
+                    textColor: textColor
+                )
+            } else {
+                CommuteNoTrainsView(textColor: textColor)
+            }
+        }
+    }
+}
+
+struct CommuteStationTitle: View {
+    let title: String
+    let destinationStation: String?
+    let textColor: Color
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            // Station name
+            Text(
+                WidgetDataFormatter().formatHeadSign(
+                    title: title,
+                    fits: { text in
+                        let textWidth = measureTextWidth(
+                            maxSize: CGSize(width: 200, height: 50),
+                            text: text,
+                            font: UIFont.arimaStyleMedium(size: 14)
+                        )
+                        return (textWidth <= 200).toKotlinBoolean()
+                    }
+                )
+            )
+            .font(Font.arimaStyleMedium(size: 14))
+            .foregroundColor(textColor)
+            .italic()
+            
+            // Destination station
+            if let destination = destinationStation {
+                HStack(spacing: 2) {
+                    Image(systemName: "arrow.right")
+                        .font(.system(size: 12))
+                        .foregroundColor(textColor)
+                    Text(destination)
+                        .font(Font.arimaStyle(size: 14))
+                        .italic()
+                        .foregroundColor(textColor)
+                }
+            }
+        }
+        .padding(.leading, 4)
+        .padding(.top, 2)
+    }
+}
+
+struct CommuteTrainDisplay: View {
+    let trains: [DepartureBoardData.TrainData]
+    let entry: CommuteProvider.Entry
+    let textColor: Color
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: -2) {
+            // First train - prominent display
+            let firstTrain = trains.first!
+            let firstTime = formatArrivalTime(firstTrain)
+            
+            HStack(alignment: .firstTextBaseline, spacing: 0) {
+                if entry.configuration.timeDisplay == .clock {
+                    // For clock time, display as single text
+                    Text(firstTime)
+                        .font(Font.arimaStyleBold(size: 32))
+                        .foregroundColor(textColor)
+                } else {
+                    // For relative time, split number and "min"
+                    let (numberPart, minPart) = splitTimeString(firstTime)
```

Comment:
This is a bit strange to have `WidgetDataFormatter` come up with a string adding the minutes text and then split it out here.

I would just extract out a `getMinutesBetween(now: Instant, time: Int): Int` in [`WidgetDataFormatter`](https://github.com/steviek/PathWidgetXplat/blob/main/composeApp/src/commonMain/kotlin/com/sixbynine/transit/path/widget/WidgetDataFormatter.kt#L141), refactor `formatRelativeTime` to use that as a helper, and use that here directly.

You should still handle the tragic, yet possible, "over an hour" case